### ISSUE TITLE
peppy.bat: quote filename for space handling, update to match current peppy.py

### ISSUE THREAD

**Issue by robmcmullen** [robmcmullen/peppy#449]
**[reported by smmckay@...]** _[Trac time 20080518 020000Z]_ If peppy.bat has spaces in its path, Python will choke unless %~f0 is quoted. Also, it needed to be updated to match peppy.py. Unfortunately, I can't see a good way to avoid putting the contents of peppy.py inside peppy.bat.

I hope this diff is useful. It doesn't look like it could be applied directly.

```
diff --git a/peppy.bat b/peppy.bat
index cb516f5..6e623e9 100644
--- a/peppy.bat
+++ b/peppy.bat
@@ -5,13 +5,18 @@ rem Don't know how this works, but it does.
 rem
 rem -------------------- DOS section --------------------
 rem You could set PYTHONPATH or TK environment variables here
-python -x %~f0 %*
+python -x "%~f0" %*
 goto exit

 """
 # -------------------- Python section --------------------
 import os,sys

+# Set default translation as a no-op for the moment (the actual translation is
+# performed on-demand during the GUI creation, not at application start time).
+import __builtin__
+__builtin__._ = unicode
+
 import peppy.main

 peppy.main.main()

```

**Comment by robmcmullen** [comment]
_[Trac time 20081007 164415Z]_ This got ticket somehow got lost in the version number reshuffling.  Back here for 0.8

**Comment by robmcmullen** [comment]
_[Trac time 20081008 025948Z]_ 1bec7e713907d8549c086356cfa0db83e63f8167 Fixed #449: peppy batch file can now properly handle spaces in install path

**Comment by robmcmullen** [comment]
_[Trac time 20080523 004231Z]_ Thanks for the patch.  I'll update the svn and include it in the next version.
